### ISSUE TITLE
Map drawing widget in plan forms

### ISSUE THREAD

**Issue by pcambra** [mstenta/farm_nfa#74]
- Zoom to assets referenced in plan
- Auto-copy shapes from referenced assets in from

**Comment by mstenta** [comment]
See https://github.com/mstenta/farm_nfa/issues/76 for Compartment creation help text.

**Comment by pcambra** [comment]
Discuss with @mstenta regarding the compartments vs locations + assets and autorefreshing the map the same way as the KML import (alpha3) works.

**Comment by pcambra** [comment]
I don't think we can manage auto refresh for MVP but the remainder would be ready to evaluate now.

**Comment by LeScruf** [comment]
I think for a first iteration of the widget, this is great. However, it doesn't seem possible to clear polygons once drawn?
